styles/main.css

```diff
@@ -14,8 +14,14 @@
   --lightBlue: #e6f7ff;
   --textColor: #23272f;
   --lightGray: #f7f6f9;
-  --mediumBlue: #CEE2FE;
+  --mediumBlue: #cee2fe;
   --darkCyan: #008b8b;
   --royalBlue: #4169e1;
   --darkMagenta: #8b008b;
+  --cardBackground: #fefffe;
+  --cardTextColor: #747574;
+  --cardSectionHoverColor: #f5f5f5;
+  --cardIcon: #666666;
+  --cardIconHover: rgba(0, 0, 0, 0.2);
+  --cardSectionActive: rgb(234, 236, 236);
```

Comment:
Try to use one color format instead of a different combination.

Reply:
yess